src/sdgio-edit.owl

```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020236>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020237>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020300>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020308>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020309>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020310>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020311>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020312>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020313>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020315>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020317>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020318>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020319>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020320>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020325>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020326>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020327>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020328>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020329>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020331>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020334>))
```

Comment:
@Mttbnchtt make an issue to import http://purl.obolibrary.org/obo/ENVO_01001170 into the next revision and add this class and related classes underneath it. The definitions would need to be revised accordingly. Definition revised.

Reply:
Done, issue #194.